src/components/BasicCard/BasicCard.tsx

```diff
@@ -0,0 +1,26 @@
+import { Card, CardActions, CardContent, Paper } from '@mui/material'
+import React, { ReactNode } from 'react'
+
+interface Props {
+    hasAction: boolean;
+    children: ReactNode;
+    sx: any;
+    element: ReactNode;
+}
+
+const BasicCard = ({ hasAction = false, children, sx = {}, element }: Props) => {

```

Comment:
Kindly use `const BasicCard: React.FC<React.PropsWithChildren<Props>>` then pass the Props 
- remove the children from your props interface since it has a built in children property in `React.PropsWithChildren`
- Once used -> don't use ({}: Props) you can directly do it like ({hasAction, ...})